File: sdkwork-app-sdk-typescript/src/types/page-map-string-object.ts
import type { PageableObject } from './pageable-object';
import type { SortObject } from './sort-object';

export interface PageMapStringObject {
  totalElements?: number;
  totalPages?: number;
  size?: number;
  content?: Record<string, unknown>[];
  number?: number;
  first?: boolean;
  last?: boolean;
  numberOfElements?: number;
  sort?: SortObject;
  pageable?: PageableObject;
  empty?: boolean;
}

File: sdkwork-app-sdk-csharp/Models/PageMapStringObject.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class PageMapStringObject
    {
        public int? TotalPages { get; set; }
        public int? TotalElements { get; set; }
        public int? Size { get; set; }
        public List<Dictionary<string, object>>? Content { get; set; }
        public int? Number { get; set; }
        public SortObject? Sort { get; set; }
        public bool? First { get; set; }
        public bool? Last { get; set; }
        public int? NumberOfElements { get; set; }
        public PageableObject? Pageable { get; set; }
        public bool? Empty { get; set; }
    }
}
